filetree/node.go

```diff
@@ -94,6 +96,9 @@ func (fn *Node) Init() {
 		case status == vcs.Stored:
 			s.Color = colors.Scheme.OnSurface
 		}
+		if fn.Info.Generated {
+			s.Color = errors.Must1(gradient.FromString("#8080C0"))
```

Comment:
This is not compatible with different color schemes...

Reply:
you got a good idea? fix it later?

Reply:
Use `colors.ToBase`.